tailnet/coordinator.go

```diff
@@ -111,16 +113,19 @@ func ServeCoordinator(conn net.Conn, updateNodes func(node []*Node) error) (func
 	}, errChan
 }
 
+const LoggerName = "coord"
```

Comment:
```suggestion
const loggerName = "coord"
```

No reason to export this.